content/en/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md

```diff
@@ -98,8 +101,7 @@ kind: KubeSchedulerConfiguration
 algorithmSource:
   provider: DefaultProvider
 
-...
-
```

Comment:
The ... indicate [elision](https://en.wikipedia.org/wiki/Elision) (&hellip). “---” is not correct here.

Reply:
reverted it back.. did this change as per reference commit.